extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -170,14 +179,14 @@ function setup_tracks_callback( \SimplePie_Item $episode ) {
 
 	// Build track data.
 	$track = array(
+		'id'          => wp_unique_id( 'podcast-track-' ),
```

Comment:
```suggestion
		'id'          => wp_unique_id( 'podcast-track' ),
```

We don't want the hyphen right?

Reply:
Why not? All other places do it

Reply:
I'm not sure. I just thought it was a typo? Why would we have a trailing hyphen when generating this ID?

I guess it doesn't matter hugely, it just stood out to me.

Reply:
The unique ID is added after that prefix (e.g. `podcast-track-1`), so I guess it makes sense to have an hyphen here.

Reply:
It felt strange the first time but I think I have done this in wpcom and is established here in Jetpack too: https://github.com/Automattic/jetpack/search?q=wp_unique_id&unscoped_q=wp_unique_id